OpenCorePkg.fdf

```diff
@@ -101,7 +101,7 @@
       GUIDED {
         DXE_DEPEX    DXE_DEPEX Optional      $(INF_OUTPUT)/$(MODULE_NAME).depex
         PE32         PE32                    $(INF_OUTPUT)/$(MODULE_NAME).efi
-        UI           STRING="$(MODULE_NAME)" Optional
+        UI           STRING="$(MODULE_NAME) $(VERSION_STRING)" Optional
```

Comment:
Hmm, if we add this, should we start actually maintaining those with our apps and drivers (most things seem to just hardcode 1.0)?

Reply:
Ah, it‘s just your GOP stuff and you actually bumped the version below - sorry

Reply:
It's currently just my GOP stuff. More generally, in theory, any other pre-OC firmware drivers we ship. Could be revisited when/if that happens, I guess.